docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -18,29 +18,29 @@ An image that {ProductName} builds from source code in a repository. One or more
 == Conftest 
 A utility for testing structured configuration data. Use Conftest to validate container information.
 
-== Enterprise Contract (EC) 
-A set of release policies that you apply to your managed environment. Use the EC to prevent the release of Supply chain Levels for Software Artifacts (SLSA) that are non-compliant. 
+== Enterprise contract (EC) 
+A set of release policies that you apply to your managed environment. Use the EC to prevent releases that are not compliant with Supply-chain Levels for Software Artifacts (SLSA) guidelines. 
 
 == Ephemeral environment
-A temporary environment that executes each integration scenario. The system creates these environments dynamically on demand and then removes them after completing the build pipeline.
+A temporary environment that executes each integration scenario. The system creates ephemeral environments dynamically on demand, and then removes them after it completes the build pipeline.
 
-== Integration test 
-A pipeline that you set up in GitHub. As you add components, it tests each one individually, and then tests the application as a whole.
+== Integration test
+A pipeline that you set up in GitHub. When you add components, it tests each one individually, and then tests the application as a whole.
 
 == Integration testing: 
 A process that uses Tekton tasks to support the setup and execution of dynamic application tests against container images.
```

Comment:
I am not crazy about the repeated term "against" in the glossary. It's a bit vague but not sure how to clarify the terms where it appears in